<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="map_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>651</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="width_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="long_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать карту</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scale_edit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="width_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>403</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ширина</string>
   </property>
  </widget>
  <widget class="QLabel" name="long_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>520</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Долгота</string>
   </property>
  </widget>
  <widget class="QLabel" name="scale_label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Масштабирование</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>550</y>
     <width>82</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>схема</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>гибрид</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>550</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>спутник</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
